backends/ebpf/tests/ptf/common.py

```diff
@@ -309,8 +318,16 @@ def _table_create_str_from_action(self, action):
         else:
             return "action name {} ".format(action)
 
-    def table_write(self, method, table, key, action=0, data=None, priority=None, references=None,
-                    counters=None, meters=None):
+    def table_write(self,
+                    method,
+                    table,
+                    key,
+                    action=0,
+                    data=None,
+                    priority=None,
+                    references=None,
+                    counters=None,
+                    meters=None):
```

Comment:
Is this always a good formatting? To me it spreads the code over too many lines, but it might still be worth the consistency.

Reply:
yapf does this in case you have named assignments. I am not sure the alternative is better. I can try to see whether there is a better configuration.